### ISSUE TITLE
When I click Edit Publii crashes

### ISSUE THREAD

**Issue by henriiquecampos** [GetPublii/Publii#572]
So...I was writing a post and I wanted to see if there was any spellchecking, so I clicked "Edit" and Publii closed...I lost my post and when I tried to see if I could reproduce basically it crashes even if I'm not editing a post no errors are shown in the terminal, this is all that I get after it crashes:
```
(Publii:4154): Gtk-WARNING **: 20:02:10.586: Theme parsing error: gtk.css:4101:29: Expected ',' in color definition

(Publii:4154): Gtk-WARNING **: 20:02:10.586: Theme parsing error: gtk.css:4142:92: The :prelight pseudo-class is deprecated. Use :hover instead.

(Publii:4154): Gtk-WARNING **: 20:02:10.586: Theme parsing error: gtk.css:4146:43: The :prelight pseudo-class is deprecated. Use :hover instead.
Segmentation fault
```
This error is printed right when Publii opens tho, not when the crash happens.

I'm using Publii 0.35.1 appimage on Solus 4.0 Gnome

**Comment by henriiquecampos** [comment]
I just checked and basically every button crashes it. I tried with File, Edit, View...every single one of them, this is very weird.

**Comment by dziudek** [comment]
Hi,

It is a similar issue to: https://github.com/GetPublii/Publii/issues/551

We will fix it in upcoming Publii v.0.35.2 in the next week :)

**Comment by dziudek** [comment]
Fixed by changes in #575
